src/integration-test/minidoc-integration.test.ts

```diff
@@ -824,7 +824,8 @@ function runTestsForBrowser(browserType: BrowserType) {
         );
       });
 
-      it('cards can be deleted', async () => {
+      // TODO: Fix this on Windows Webkit and re-enable the test.
+      xit('cards can be deleted', async () => {
```

Comment:
I say we disable Windows Webkit for now, instead of disabling this test. What do you think?

Reply:
Makes sense! While trying to exclude webkit tests on Windows, they somehow succeeded :) Not sure what was wrong yesterday.

![Screen Shot 2021-03-09 at 14 38 45](https://user-images.githubusercontent.com/2227041/110465188-287bcc00-80e5-11eb-9e7d-8967ff07aca8.png)